list.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'enumerable'
```

Comment:
- [ ] Kindly make sure you spell the file's name correctly. You may notice you use `enumerable` whereas your file's name is `myenumerable`. Make sure you fix that.